docs/user-guide/pods/init-container.md

```diff
@@ -0,0 +1,169 @@
+---
```

Comment:
This file should be renamed `init-containers.md` to match the link in `_data/guides.yml`. The sidenav link is currently broken in the Netlify preview.